src/index.js

```diff
@@ -223,7 +223,18 @@ function querySelectorAll (selector, context = document) {
   return query(selector, context, true)
 }
 
+function getElementsByClassName (classNames, context = document) {
+  const oneSpaceSeparator = classNames.trim().replace(/\s+/g, ' ');
+  const classNamesSplit = oneSpaceSeparator.split(' ');
+  let selector = "";
+  for (let i = 0; i < classNamesSplit.length; i++) {
+    selector += '.' + classNamesSplit[i];
+  }
```

Comment:
Just occurred to me: instead of building up a class selector string, maybe we could use `ElementIterator` instead? I.e. iterate through the elements and check their `.classList`. The benefit is that, for folks who want to just use `getElementsByClassName`, they don't need to import all of `postcss-selector-parser` (it can be tree-shaken out).